python/lsst/pipe/tasks/fakes.py

```diff
@@ -60,8 +60,9 @@ class constructor.  They can then assume self.config is an instance of their Con
         multiple times by the run() method.
         """
         lsst.pipe.base.Task.__init__(self, **kwargs)
-        lsst.afw.image.MaskU.addMaskPlane(self.config.maskPlaneName)
-        self.bitmask = lsst.afw.image.MaskU.getPlaneBitMask(self.config.maskPlaneName)
+        lsst.afw.image.Mask[lsst.afw.image.MaskPixel].addMaskPlane(self.config.maskPlaneName)
+        self.bitmask = lsst.afw.image.Mask[lsst.afw.image.MaskPixel]\
+            .getPlaneBitMask(self.config.maskPlaneName)
```

Comment:
I think if you put the period on the line before, you don't need the (evil) backslash.
But the line will shorten when we remove the `[lsst.afw.image.MaskPixel]`.